### ISSUE TITLE
Bug: Radio Buttons should not be wrapped in a `<ul>`

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#596]
Currently fails a11y test:

![image](https://user-images.githubusercontent.com/1212885/181177763-6a9728f5-3efc-43c1-bc78-f7d43ce390b4.png)

I suggest to change the `<ul>` to a `<div>`
